### ISSUE TITLE
Repeating the same word in a string may produce a wrong result

### ISSUE THREAD

**Issue by mrkevinze** [devongovett/regexgen#5]
Can the input strings contain multiple words? I was presuming that they can, so I started trying it out.

This seems ok.
```
./cli.js "hi" "hi friend"
/hi(?: friend)*/
```

But this does not.
```
./cli.js "hey" "hi" "hi friend"
/h(?:i|ey)/
```

**Comment by devongovett** [comment]
Hmm, first one seems wrong too. Should be `?` instead of `*` there. I think there is a bug in the minimizer. Taking a look.

**Comment by devongovett** [comment]
Fixed in 72ef1c26ef73f3358b90c696f4ff7ce4782d9f13. Released in v1.2.0.
